test/snapshots/prefer-error-cause.mjs.md

```diff
@@ -0,0 +1,1907 @@
+# Snapshot report for `test/prefer-error-cause.mjs`
+
+The actual snapshot is saved in `prefer-error-cause.mjs.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## Invalid #1
+      1 | try {} catch (error) { throw new Error(); }
+
+> Error 1/1
+
+    `␊
+    > 1 | try {} catch (error) { throw new Error(); }␊
+        |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer specifying an old error as \`cause\` option when rethrowing the error.␊
```

Comment:
Let's report on the new error.

Reply:
I think to implement reporting correctly `new Error` line when fix is possible, I need to extract line number of given Node (and also column start and end) for passing `loc` argument to `context.report`.
By any chance, are there any utility functions for this sort of purpose?